Mathlib/MeasureTheory/Measure/MeasureSpaceDef.lean

```diff
@@ -235,12 +240,14 @@ theorem measure_union_lt_top_iff : μ (s ∪ t) < ∞ ↔ μ s < ∞ ∧ μ t <
   · exact (measure_mono Set.subset_union_left).trans_lt h
   · exact (measure_mono Set.subset_union_right).trans_lt h
 
+@[aesop (rule_sets := [finiteness]) safe apply]
 theorem measure_union_ne_top (hs : μ s ≠ ∞) (ht : μ t ≠ ∞) : μ (s ∪ t) ≠ ∞ :=
   (measure_union_lt_top hs.lt_top ht.lt_top).ne
 
 open scoped symmDiff in
+@[aesop (rule_sets := [finiteness]) unsafe apply]
```

Comment:
I'm wondering if we should mark this as safe even though it's unsafe. It's definitely likely to work. It should have at least a very high percentage, so maybe it should be `unsafe 95% apply`?